aws/src/main/java/org/apache/iceberg/aws/AssumeRoleAwsClientFactory.java

```diff
@@ -46,12 +46,14 @@
   private int timeout;
   private String region;
   private String s3Endpoint;
+  private Boolean s3UseArnRegionEnabled;
```

Comment:
Why is this boxed?

Reply:
Thanks, made the changes.

I overlooked it, I think `primitive` should be sufficient.